src/main/java/io/cdap/plugin/servicenow/source/apiclient/ServiceNowTableAPIClientImpl.java

```diff
@@ -102,6 +104,37 @@ public String getAccessToken() throws OAuthSystemException, OAuthProblemExceptio
     }
   }
 
+  /**
+   * Create a new record in the ServiceNow Table
+   *
+   * @param tableName ServiceNow Table name
+   * @param entity Details of the Record to be created
+   */
+  public void createRecord(String tableName, HttpEntity entity) {
+    ServiceNowTableAPIRequestBuilder requestBuilder = new ServiceNowTableAPIRequestBuilder(
+      this.conf.getRestApiEndpoint(), tableName);
+
+    RestAPIResponse apiResponse = null;
+
+    try {
+      String accessToken = getAccessToken();
+      requestBuilder.setAuthHeader(accessToken);
+      requestBuilder.setAcceptHeader("application/json");
+      requestBuilder.setContentTypeHeader("application/json");
+      requestBuilder.setEntity(entity);
+      apiResponse = executePost(requestBuilder.build());
+      if (!apiResponse.isSuccess()) {
+        LOG.error("Error - {}", getErrorMessage(apiResponse.getResponseBody()));
+      } else {
+        LOG.info(apiResponse.getResponseBody().toString());
+      }
+    } catch (OAuthSystemException | OAuthProblemException e) {
+      LOG.error("Error in creating a new record", e);
+    } catch (UnsupportedEncodingException e) {
+      e.printStackTrace();
```

Comment:
write to logger. Also we should throw exception so that the test fails on error

Reply:
Added.